apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/OpenTextSummary.tsx

```diff
@@ -72,6 +79,16 @@ export default function OpenTextSummary({ questionSummary, environmentId }: Open
             </div>
           );
         })}
+
+        <div className="my-1 flex justify-center">
```

Comment:
Can you please increase the vertical margin a bit so that it equivalent to the size rows. It would improve the UI a bit.
<img width="696" alt="Screenshot 2023-10-23 at 6 31 25 PM" src="https://github.com/formbricks/formbricks/assets/31568159/46d4bd7b-9216-4967-a84f-a355c7557a5a">
<img width="698" alt="Screenshot 2023-10-23 at 6 31 37 PM" src="https://github.com/formbricks/formbricks/assets/31568159/dfcb5bf6-a8fe-47d9-a015-fe513d608491">